deploy-commands.js

```diff
@@ -1,18 +1,18 @@
-const { SlashCommandBuilder, SlashCommandSubcommandBuilder } = require('@discordjs/builders');
```

Comment:
No Changes in this file, just eslint formatting. You can ignore all the "changes".